packages/records/lib/db/migrations/20240327151027_create_records_table.ts

```diff
@@ -23,7 +23,6 @@ export async function up(knex: Knex): Promise<void> {
             deleted_at timestamp with time zone,
             sync_id uuid,
             sync_job_id integer,
-            CONSTRAINT ${TABLE}_connection_id_external_id_model UNIQUE (connection_id, external_id, model)
```

Comment:
was defined twice. Here and https://github.com/NangoHQ/nango/pull/1996/files#diff-b8bc3ca28e369e9030e8fb23d563c153a1fc8b88c11d08083b1ed2ed3a57c445R73

Reply:
the one you highlighted is not UNIQUE tho

Reply:
yes but somehow a unique constraint is still being created. Here is the DDL when running the migration locally 
```
CREATE TABLE nango_records.records (
    id uuid NOT NULL,
    external_id character varying(255) NOT NULL,
    json jsonb,
    data_hash character varying(255) NOT NULL,
    connection_id integer NOT NULL,
    model character varying(255) NOT NULL,
    created_at timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
    updated_at timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
    deleted_at timestamp with time zone,
    sync_id uuid,
    sync_job_id integer,
    CONSTRAINT records_connection_id_external_id_model UNIQUE (connection_id, external_id, model),
    CONSTRAINT records_connection_id_model_id_unique UNIQUE (connection_id, model, id)
);

-- Indices -------------------------------------------------------

CREATE UNIQUE INDEX records_connection_id_external_id_model ON nango_records.records(connection_id int4_ops,external_id text_ops,model text_ops);
CREATE INDEX records_connection_id_model_external_id_index ON nango_records.records(connection_id int4_ops,model text_ops,external_id text_ops);
CREATE INDEX records_connection_id_model_updated_at_id_index ON nango_records.records(connection_id int4_ops,model text_ops,updated_at timestamptz_ops,id uuid_ops);
CREATE INDEX records_sync_id_index ON nango_records.records(sync_id uuid_ops);
CREATE INDEX records_sync_job_id_index ON nango_records.records(sync_job_id int4_ops);
CREATE UNIQUE INDEX records_connection_id_model_id_unique ON nango_records.records(connection_id int4_ops,model text_ops,id uuid_ops);
```

Reply:
There is a syntax error but otherwise looks good now 👍🏻